docs/source/usage/tutorials/Deep/Double_DQN.rst

```diff
@@ -0,0 +1,98 @@
+=====================
+Double Deep Q-Network
+=====================
+
+Objective
+=========
+
+Double DQN builds upon the notion of Double Q-Learning and extends it to Deep Q-networks. The main objective is to resolve the problem of overestimation of Q-values. This is done by defining the traget q-values in a slightly differen manner. The loss function is defined as:
```

Comment:
Can you explain here why is there an overestimation of Q-values in a vanilla DQN?

Reply:
Cool, will add this part to every tutorial